packages/fast-form-generator-react/src/form/form-section.style.ts

```diff
@@ -40,17 +44,19 @@ const styles: ComponentStyles<IFormSectionClassNameContract, {}> = {
                 right: "0"
             },
             "& a": {
-                display: "block",
-                ...localizePadding(12, 18, 12, 1)
```

Comment:
you can remove the import for this since it looks like we aren't using it in the file any longer